spec/models/miq_worker/container_common_spec.rb

```diff
@@ -113,6 +113,35 @@ def deployment_name_for(name)
         expect(klass.constantize.new.worker_deployment_name.length).to be <= 60
       end
     end
+
+    context "ems_id" do
+      let(:subject) { ManageIQ::Providers::Openshift::ContainerManager::EventCatcher.new }
```

Comment:
Does this have to be an `Openshift::ContainerManager::EventCatcher` for the specs?  Would prefer not to call out a specific provider if possible.  Could you use `ManageIQ::Providers::BaseManager::EventCatcher` ?